### PR TITLE
Update content-type.txt

### PR DESCRIPTION
**Pull request by A1vinSmith**

### CODE REVIEW

**Comment by g0tmi1k** [comment]
Source?

**Comment by A1vinSmith** [comment]
Not sure why. But something like this https://cweiske.de/tagebuch/php-mimetype.htm

**Review by Sandesh333333 — CHANGES_REQUESTED** [review]

File: Miscellaneous/web/content-type.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+application/x-php
 application/vnd.hzn-3d-crossword
 video/3gpp
 video/3gpp2
```
[the remark this answers is not in the document]
Comment (A1vinSmith) [reply Miscellaneous/web/content-type.txt:4]: sorry what do you mean by that? Adding a comment for the line?

**Review by PinkDev1 — APPROVED** [review]

**Comment by g0tmi1k** [comment]
Like to try and give credit where possible. also helps people to understand more/

**Comment by g0tmi1k** [comment]
Thank you!

**Review by Sandesh333333 — APPROVED** [review]

**Pull request merged**